packages/edit-widgets/src/components/widget-areas-block-editor-content/style.scss

```diff
@@ -1,6 +1,9 @@
 .edit-widgets-block-editor {
 	position: relative;
 
+	// This is the default font that is going to be used in the content of the areas (blocks).
+	font-family: $default-font;
```

Comment:
I think something we can consider later in the `block-editor` package (so not WP but more generic) is to use "serif" font if there are no "styles" setting provided.